lib/membrane_vpx/encoder/vp8_encoder.ex

```diff
@@ -36,6 +36,20 @@ defmodule Membrane.VP8.Encoder do
                 of the incoming stream. Some reference recommended bitrates can be also found
                 [here](https://support.google.com/youtube/answer/1722171#zippy=%2Cbitrate)
                 """
+              ],
+              g_lag_in_frames: [
+                spec: non_neg_integer(),
+                default: 5,
```

Comment:
is this one the enoder's default or was it just your choice?

Reply:
my choice, the default was 25